builtin/mainmenu/tab_online.lua

```diff
@@ -167,6 +170,7 @@ local function main_button_handler(tabview, fields, name, tabdata)
 				if gamedata.address and gamedata.port then
 					core.settings:set("address", gamedata.address)
 					core.settings:set("remote_port", gamedata.port)
+					core.settings:set("server_description", gamedata.serverdescription or "")
```

Comment:
This isn't needed, please remove

Reply:
Sorry i forgot : "PS : Merge PR : #7316 and #7308 before this one"

Reply:
how is that relevant here?
you should be making sure that your git history is clean (not cluttered with unrelated changes)

Reply:
Because i rebase my branches. I need his changes to test. Otherwise it will crash